### ISSUE TITLE
Set up DoSOCSv2 Wrapper Flask server

### ISSUE THREAD

**Issue by ccarterlandis** [computationalmystic/augur-group13#2]
**Type:** Structural

**Purpose:**
To use DoSOCSv2 we need to wrap it in a Flask REST API we create. The REST API is to be defined by issue #3. The purpose of this wrapper is to have an Augur DoSOCSv2 plugin request dosocs information and ask the wrapper for that information.

**Step in overall process (Bold item is the item this issue aims to solve):** 
1. Request git URL through Augur
2. **Augur DoSOCSv2 plugin requests DoSOCS information from the DoSOCSv2 Wrapper Flask Server**
3. Augur DoSOCSv2 plugin takes returned information and parses it into the two UI enum fields (#5)
We need to set up our Flask server so that we can build an API with it. Operationally, the server =/= the API, so I feel it's at least somewhat pertinent to separate these efforts. This server should essentially just be routing requests from the API to it's endpoints.

**Progress Update (5/12/19):** API is being worked on, thus partially blocked by #3. @Pauldg7 has gone ahead and written a wrapper to take a git url and return the response of the DoSOCSv2 oneshot command (This command scans a package, generates a document, and prints an SPDX document in one shot).

**Comment by ccarterlandis** [comment]
AFAIK, @Luke-Fisher has been making steady progress on this. Using an old project as a template, we have been to able to start laying the groundwork by setting up a rudimentary web server.

**Comment by Luke-Fisher** [comment]
Here's a summary of the progress on the flask server from Thursday and Friday:

Thursday: I started working off of @ccarterlandis 's code and familiarizing myself with Python and Flask. The server I made for my earlier tests and proof of concept was more or less be discarded in favor of using Carter's better structured code. 

Friday: I struggled with getting the server (and related code) to work together properly. I changed a few too many things too fast without fully understanding how they worked and now the code's pretty broken. The bulk of Friday's time on this issue was just rolling back broken code and reading documentation.
